legacy/jacorb/src/main/java/org/jboss/as/jacorb/JacORBSubsystemDefinitions.java

```diff
@@ -139,7 +140,6 @@
     // connection attribute definitions.
     public static final SimpleAttributeDefinition ORB_CONN_RETRIES = new SimpleAttributeDefinitionBuilder(
             JacORBSubsystemConstants.ORB_CONN_RETRIES, ModelType.INT, true)
-            .setDefaultValue(new ModelNode().set(5))
```

Comment:
These default changes are a problem. It means a jacorb config that isn't used for EAP7 servers, but is used for EAP6 will now have different behavior. That's ok if the change is really necessary, but it's best avoided.

Actually, it's worse than that. Because the domain controller will say there's no default (or in the case of some other props, say that the default is "X") but then a legacy slave will see an undefined node and use "5", since it's code says "5". So if you change a default, you have to add a transformer such that if the param is undefined, you change the value to the new default.